projects/users/src/favorites-service.ts

```diff
@@ -0,0 +1,29 @@
+import { UpdateResult } from 'mongodb';
+import { UsersService } from './users-service';
+
+export interface FavoriteUpdateOptions {
+  userId: string,
+  soundId: string,
```

Comment:
I tend to use semicolons in interfaces, but you can use anything and it doesn't produce an error cuz interfaces aren't real.

```suggestion
  userId: string;
  soundId: string;
```

Reply:
It lets them through because interfaces are a TypeScript thing, so they don't get compiled to JavaScript, they're just there for type checking.

Reply:
yeah I know that part but usually eslint will yell